WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,16 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read("app/views/search_results/wca-regulations.json"))
+  rescue
+    REGULATIONS = [].freeze
+  end
+
+  def limit(number)
+    __getobj__.first(number)
```

Comment:
Do we need `__getobj__` here? I think that's what SimpleDelegator should handle.